src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeWhatsNew.xaml.cs

```diff
@@ -91,6 +92,7 @@ private bool GetShowDataDiagnosticsInfoBar()
         /// </summary>
         private const string RemoveInstallerHashesRegex = @"(\r\n)+## Installer Hashes(\r\n.*)+## Highlights";
         private const string RemoveHotFixInstallerHashesRegex = @"(\r\n)+## Installer Hashes(\r\n.*)+$";
+        private const string GitHubCurrentReleasePlaceholder = "github-current-release-work";
```

Comment:
Can we please be more specific with the replace? I'd prefer if we replace the whole "[github-current-release-work]" with "[github-current-release-work{counter}]".